repo/packages/A/arangodb/0/marathon.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "{{arangodb/id}}",
+  "cpus": 0.2,
```

Comment:
cpu / mem and all resources must be configurable.

example: https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/M/marathon/1/marathon.json